src/dnddiceroller/DiceTower.java

```diff
@@ -17,12 +19,33 @@
 public class DiceTower {
   final int PANEL_COUNT = 3;
   List<Die> dice;
+  int trayValue;
 
   public DiceTower() {
 	this.dice = new ArrayList();
+        int trayValue=0;
```

Comment:
Do I need to keep this class then.

Zachary T. Murphy
Computer Technology Major
WCCC

Mobile: 724-516-7201

Sent from my iPhone

On Oct 20, 2015, at 3:41 PM, Paul Scarrone <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/dnddiceroller/DiceTower.java<https://github.com/WCCCEDU/CPT-163-27-F2015-Assignment-7-Array-DiceTower/pull/3#discussion_r42543298>:

>
>    public DiceTower() {
>       this.dice = new ArrayList();
> +        int trayValue=0;


you are re-initializing trayValue in the constructors scope. It does not assign the field.

-
Reply to this email directly or view it on GitHub<https://github.com/WCCCEDU/CPT-163-27-F2015-Assignment-7-Array-DiceTower/pull/3/files#r42543298>.